include/seqan3/core/algorithm/detail/blocking_algorithm_executor.hpp

```diff
@@ -293,14 +265,14 @@ class blocking_algorithm_executor
      * Clears all buckets and sets the bucket iterator to the first iterator, such that the allocated memory for each
```

Comment:
probably same as above: doesn't this set the `buffer_it` and not the `bucket_iterator` to the first bucket in the buffer and clears all buckets ?